### ISSUE TITLE
Video block has an aria-label on the div

### ISSUE THREAD

**Issue by troyrutter** [iastate/iastate22-wordpress#46]
Apparently div's aren't supposed to have aria labels.   Giving the following accessibility issue in site improve:

An ARIA attribute has been specified that is either not supported or is prohibited, on this type of element.

<img width="1051" alt="Screenshot 2024-11-22 at 3 29 27 PM" src="https://github.com/user-attachments/assets/2c7b4add-30de-46be-a351-ae4f2e17989f">

**Comment by troyrutter** [comment]
according to https://www.davidmacd.com/blog/does-aria-label-override-static-text.html#:~:text=By%20design%2C%20aria%2Dlabel%20or,ignored%20by%20VoiceOver%20on%20iOS).

DIVs should only have aria labels if they have role=navigation, search, main, img

Based on this, the aria-label on the video embed div should be removed.
